src/main/proto/ga4gh/rna_quantification.proto

```diff
@@ -70,25 +86,30 @@ message ExpressionLevel {
   // Name
   string name = 2;
 
-  // The associated QuantificationGroup. 
-  string quantification_group_id = 3;
+  // Id of the feature
+  string feature_id = 3;
+
+  // The associated ExpressionLevelGroup. 
+  string expression_level_group_id = 4;
```

Comment:
Since we want to remove `ExpressionLevelGroup` ASAP, replacing it with a more appropriate entity from the sequence annotation API, I would like to propose that we not use it as a critical link between individual `ExpressionLevel`s and `RnaQuantifcation`s. To this end, I suggest we have an `rna_quantification_id` here instead.